containers/forwarder/Dockerfile

```
@@ -1,3 +1,8 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.13 AS builder
+COPY audit-filter /go/src/audit-filter
+WORKDIR /go/src/audit-filter
+RUN make
```

Suggest being explicit about the target, so Makefile edits don't have unintended repercussions.
```suggestion
RUN make build
```